app/controllers/targets_controller.rb

```diff
@@ -9,6 +9,7 @@ def new
   def index
     @all_targets = Target.all
     @targets = @all_targets.last(10)
```

Comment:
Are you using the global var `@targets` somewhere? Else you can delete it

Reply:
I use it in the index view for rendering last 10 targets